packages/button/README.md

```diff
@@ -1 +1,115 @@
-# button
+![Telegraph by Knock](https://github.com/knocklabs/telegraph/assets/29106675/9b5022e3-b02c-4582-ba57-3d6171e45e44)
+
+[![npm version](https://img.shields.io/npm/v/@telegraph/button.svg)](https://www.npmjs.com/package/@telegraph/button)
+
+# @telegraph/button
+> A button component
+
+## Installation Instructions
+
+```
+npm install @telegraph/button
+```
+
+### Add stylesheet
+```
+@import "@telegraph/button"
+OR
+@import "@telegraph/button/scoped"
```

Comment:
Left a related question re: this here: https://github.com/knocklabs/telegraph/pull/48#discussion_r1491503130